hack/verify-shellcheck.sh

```diff
@@ -0,0 +1,170 @@
+./hack/update-generated-completions.sh
+./hack/verify-open-ports.sh
+./hack/test-cmd.sh
+./hack/export-certs.sh
+./hack/test-go.sh
+./hack/test-integration.sh
+./hack/build-assets.sh
+./hack/test-rpm.sh
+./hack/test-tools.sh
+./hack/verify-upstream-commits.sh
+./hack/install-std-race.sh
+./hack/extract-release.sh
+./hack/build-in-docker.sh
+./hack/make-p12-cert.sh
+./hack/common.sh
+./hack/pythia.sh
+./hack/move-upstream.sh
+./hack/update-generated-swagger-spec.sh
+./hack/build-go.sh
+./hack/rebase-describe-bumps.sh
+./hack/test-cmd_util.sh
+./hack/verify-generated-swagger-spec.sh
+./hack/install-tools.sh
+./hack/verify-gofmt.sh
+./hack/update-generated-swagger-descriptions.sh
+./hack/verify-generated-conversions.sh
+./hack/convert-samples.sh
+./hack/verify-generated-deep-copies.sh
+./hack/cherry-pick.sh
+./hack/dind-cluster.sh
+./hack/verify-generated-completions.sh
+./hack/update-generated-conversions.sh
+./hack/verify-generated-docs.sh
+./hack/update-generated-docs.sh
+./hack/push-release.sh
+./hack/update-generated-deep-copies.sh
+./hack/install-assets.sh
+./hack/lib/util/environment.sh
+./hack/lib/log.sh
+./hack/test-end-to-end-docker.sh
+./hack/test-assets.sh
+./hack/build-images.sh
+./hack/serve-local-assets.sh
+./hack/verify-generated-swagger-descriptions.sh
+./hack/build-base-images.sh
+./hack/update-external-examples.sh
+./hack/verify-golint.sh
+./hack/util.sh
+./examples/zookeeper/teardown.sh
+./examples/zookeeper/config-and-run.sh
+./examples/project-spawner/project-spawner.sh
+./examples/data-population/services.sh
+./examples/data-population/quotas.sh
+./examples/data-population/populate.sh
+./examples/data-population/projects.sh
+./examples/data-population/templates.sh
+./examples/data-population/common.sh
+./examples/data-population/apps.sh
+./examples/data-population/users.sh
+./examples/data-population/limits.sh
+./examples/etcd/etcd-discovery.sh
+./examples/etcd/etcd.sh
+./examples/sample-app/pullimages.sh
+./examples/sample-app/cleanup.sh
+./images/node/scripts/origin-node-run.sh
+./images/openldap/test-init.sh
+./images/openldap/init.sh
+./images/openvswitch/scripts/ovs-run.sh
+./images/release/openshift-origin-build.sh
+./images/ipfailover/keepalived/tests/verify_failover_image.sh
+./images/ipfailover/keepalived/conf/settings.sh
+./images/ipfailover/keepalived/monitor.sh
+./images/ipfailover/keepalived/lib/utils.sh
+./images/ipfailover/keepalived/lib/config-generators.sh
+./images/ipfailover/keepalived/lib/failover-functions.sh
+./images/builder/docker/custom-docker-builder/build.sh
+./contrib/vagrant/provision-config.sh
+./contrib/vagrant/provision-node.sh
+./contrib/vagrant/provision-util.sh
+./contrib/vagrant/provision-master.sh
+./contrib/vagrant/provision-minimal.sh
+./contrib/vagrant/provision-full.sh
+./contrib/vagrant/provision-dind.sh
+./contrib/node/install-sdn.sh
+./tools/junitreport/test/integration.sh
+'
+for file in ${files}; do
+    if ! echo "${blacklist}" | grep -q "${file}"; then
```

Comment:
Oh.  That makes me sad.

Another idea is to put the blacklist into a file `blacklist.txt` and use `grep -f`.  Or use `comm`, but that's probably too advanced if the performance cost of `grep` in a `for` loop is negligible.